app/includes/header.test.js

```diff
@@ -112,5 +112,25 @@ describe('header', () => {
         });
       }));
     });
+
+    it('should render the covid19 global warning', createTestHarness(setup, (harness) => {
```

Comment:
should have another test for should not show too (or add this to another test to make sure the banner is not shown)